platform/android/src/native_map_view.cpp

```diff
@@ -858,7 +858,6 @@ jni::Array<jni::Object<Source>> NativeMapView::getSources(JNIEnv& env) {
     for (auto source : sources) {
         auto jSource = jni::Object<Source>(createJavaSourcePeer(env, *rendererFrontend, *source));
         jSources.Set(env, index, jSource);
-        jni::DeleteLocalRef(env, jSource);
```

Comment:
Should this actually be removed? `jSource` is still a local reference here right? In practice there will not be that many sources, but this could potentially overflow the local reference table I think.

Reply:
This is now a pointer to the global reference from `GetJavaPeer`.